src/components/HeadCellMenu/MenuList/MenuGroup.tsx

```diff
@@ -63,18 +63,19 @@ const MenuGroupItems = ({
   return (
     <>
       <StyledMenuItem
-        ref={subMenus ? anchorRef : null}
+        ref={subMenus ? (anchorRef as React.RefObject<HTMLLIElement>) : null}
         key={id}
         data-testid={`menu-item-${id}`}
         className="nebula-table-utils-head-menu-item"
+        // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
I actually fixed this no-missused-promised ts error based on onclicks yesterday. it was because of return type of onclick callbacks and [awaiting some promise based functions](https://github.com/qlik-oss/nebula-table-utils/pull/20/commits/cab250f552ce776d49cadbacead7e49734f68b48) while that is being passed from host repos (SNT and PVT) but not sure why `@qlik/tsconfig/react.json` expects it as error again in this case 🥲

Reply:
Still seem to be there